### PR TITLE
chore: added the MsgsetDenomMetadata in cli

### PR DESCRIPTION
**Pull request by vishal-kanna**
closes: #7941

### CODE REVIEW

File: x/tokenfactory/client/cli/tx.go [hunk 1]
```diff
@@ -21,11 +21,19 @@ func GetTxCmd() *cobra.Command {
...
 		NewSetBeforeSendHookCmd(),
+		NewMsgSetDenomMetadata(),
 	)
 
 	return cmd
 }
 
+func NewMsgSetDenomMetadata() *cobra.Command {
+	return osmocli.BuildTxCli[*types.MsgSetDenomMetadata](&osmocli.TxCliDesc{
+		Use:   "set denom metadata",
```
Comment (PaddyMc) [inline x/tokenfactory/client/cli/tx.go:32]: thanks for the PR, can we change this to "set-denom-metadata"?

**Review by PaddyMc — CHANGES_REQUESTED** [review]

File: x/tokenfactory/client/cli/tx.go (hunk 1 above)
Comment (vishal-kanna) [reply x/tokenfactory/client/cli/tx.go:32]: ok @PaddyMc

**Review by mattverse — APPROVED** [review]
LGTM

**Review by nicolaslara — APPROVED** [review]

**Review by PaddyMc — APPROVED** [review]

**Comment by PaddyMc** [comment]
Merged ty! :bow:

**Pull request merged**